bin/nodenv-each

```diff
@@ -23,6 +25,8 @@ if [ "$1" == --complete ]; then
   exit
 fi
 
+jobs=$(nproc)
```

Comment:
I'd like to keep the nproc call isolated so that this subshell is only spawned when a user requests parallelism.